src/modules/users/edit-user/edit-user.js

```diff
@@ -99,8 +96,10 @@
         function _validateUIFields() {
             var isFormValid = true,
                 form = vm.editUserForm;
-
-            if (form.name.$invalid) {
+            if(form.name.$dirty && form.name.$error.maxlength){
+                vm.errorMsg = "Name is too long (maximum 20 characters).";
+                isFormValid = false;
+            } else if (form.name.$invalid) {
```

Comment:
Is there a case where "form.name.$invalid" is reached?

Reply:
@umangachapagain why you think it will not? 

I think when user will try to submit form without name it will show this error? what you think

Reply:
can user submit form without name?
Because we have minlength of 4 and maxlength of 20, whenever user types anything smaller than 4 characters, i get an inbuilt error prompt that says "Extend name to 4 characters" and when its longer than 20 characters, we throw a "Name is too long" error.

any thing other than minlength and maxlength could be caught here, but as I see it, there are no other restrictions on name. I can even write a whole script as my name and it will accept it as long as length requirement is satisfied.